README.md

```diff
@@ -17,9 +17,12 @@ $ npm install --save multihashes # node the name of the module is multihashes
 var multihashes = require('multihashes')
 ```
 
-### In the Browser through browserify
+### Browser: Browserify, Webpack, other bundlers
+
+The code published to npm that gets loaded on require is in fact a ES5 transpiled version with the right shims added. This means that you can require it and use with your favourite bundler without having to adjust asset management process.
+
+```var multihash = require('js-multihash')
 
-Same as in Node.js, you just have to [browserify](https://github.com/substack/js-browserify) the code before serving it. See the browserify repo for how to do that.
 
 ### In the Browser through `<script>` tag
 
```

Comment:
This section needs to be updated to something like: https://github.com/ipfs/js-ipfs-blocks#browser-script-tag